src/components/BigButton.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+interface BigButtonProps {
+  color?: string;
+  onChange?: () => void;
+}
+
+const BigButton = styled.button<BigButtonProps>`
+  background-color: ${props => props.color || 'blue'};
+  color: white;
+  padding: 10px 20px;
+  border: none;
+  border-radius: 5px;
```

Comment:
Flagging this here as well for `rem` unit conversion! `border-radius` is one of those things that can get finicky, but I think we do want it to scale with `font-size`.